### DISCUSSION TITLE
How to bold only select y-axis labels?

### DISCUSSION THREAD

**Discussion by WuSelina** [uclahs-cds/public-R-BoutrosLab-plotting-general#108] [Q&A] [answered]
Hi all,
I am working with `create.heatmap()` and have gene names along the y-axis and samples along the x-axis. There are some genes I would like to bold, and I am wondering how the community does this, if done so before. I have tried `substitute(paste(bold(gene.name)))` as the values for gene names, but this whole string ends up being plotted.

For example, what if I wanted to bold only 'MBOAT2', 'MUC17', and 'NFKB2'?
![image](https://user-images.githubusercontent.com/68160979/226244246-d0317b06-eb80-4025-87d5-fa08d3750c6a.png)

Thank you so much!

**Comment by pboutros** [comment]
This should be doable using the `xaxis.fontface` parameter, but I'll create an issue to better document which things do/don't take vectors, add an example of vector recycling to the vignette and to improve ordering of variables in documentation.

**Comment by WuSelina** [comment, marked as answer]
I used `yaxis.fontface` parameter as directed and gave it a vector as follows:
```
yaxis.fontface = ifelse(
        df$genes %in% genes.to.bold,
        'bold',
        'plain'
        )
```
